{
  "Faults": [
    {
      "latitude": 51.49456380846527,
      "type": "HV",
      "reference": "KX1949",
      "etaUpdate": true,
      "affectedCustomerCount": 8,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "20abf1ca-73c5-4228-ad3b-0f70bb388427",
      "minLongitude": -0.6035,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5984833776666956,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T10:45:00.000+0000",
      "estimatedRestoration": "2023-05-20T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX1949'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX1949",
      "maxLongitude": -0.5855,
      "minLatitude": 51.4851,
      "name": "SL4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL4 6HW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6035,
              51.4991
            ],
            [
              -0.5855,
              51.4991
            ],
            [
              -0.5855,
              51.4851
            ],
            [
              -0.6035,
              51.4851
            ],
            [
              -0.6035,
              51.4991
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T11:10:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4991,
      "etrUpdate": true
    },
    {
      "latitude": 51.79275616842682,
      "type": "HV",
      "reference": "KX2008",
      "etaUpdate": true,
      "affectedCustomerCount": 246,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7e36c745-1d2e-40e2-a6a8-35e87fb6f2ce",
      "minLongitude": -1.3676,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3360583640229051,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T10:14:00.000+0000",
      "estimatedRestoration": "2023-05-20T18:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2008'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2008",
      "maxLongitude": -1.2595,
      "minLatitude": 51.7534,
      "name": "OX29 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX1 2AU",
        "OX1 2AY",
        "OX29 4BN",
        "OX29 4BU",
        "OX29 4BW",
        "OX29 4DG",
        "OX29 4DL",
        "OX29 4DN",
        "OX29 4DP",
        "OX29 4DR",
        "OX29 4DS",
        "OX29 4DT",
        "OX29 4DW",
        "OX29 4DX",
        "OX29 4DY",
        "OX29 4DZ",
        "OX29 4EA",
        "OX29 4EB",
        "OX29 4EL",
        "OX29 4EN",
        "OX29 4EP",
        "OX29 4ES",
        "OX29 4ET",
        "OX29 4SU",
        "OX29 4SZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3676,
              51.8171
            ],
            [
              -1.2595,
              51.8171
            ],
            [
              -1.2595,
              51.7534
            ],
            [
              -1.3676,
              51.7534
            ],
            [
              -1.3676,
              51.8171
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T10:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "48fe9db2-3246-4831-94d0-56f223aad601",
        "6fc91e36-d085-4c30-a14a-078adab6567e",
        "12fc7655-e95c-4cbf-b769-2a742676f87c"
      ],
      "maxLatitude": 51.8171,
      "etrUpdate": true
    },
    {
      "latitude": 51.22848864778126,
      "type": "LV",
      "reference": "KX2031",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-20T14:35:00.000+0000",
      "affectedCustomerCount": 21,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "05c1ebd9-67d1-4e46-b590-f87577204434",
      "minLongitude": -1.4799,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.475892288102064,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T12:20:00.000+0000",
      "estimatedRestoration": "2023-05-20T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2031'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2031",
      "maxLongitude": -1.4733,
      "minLatitude": 51.2273,
      "name": "SP10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP10 5NA",
        "SP10 5ND",
        "SP10 5NG",
        "SP10 5NJ",
        "SP10 5NL",
        "SP10 5NQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4799,
              51.2313
            ],
            [
              -1.4733,
              51.2313
            ],
            [
              -1.4733,
              51.2273
            ],
            [
              -1.4799,
              51.2273
            ],
            [
              -1.4799,
              51.2313
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T15:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2313,
      "etrUpdate": true
    },
    {
      "latitude": 51.44270139551498,
      "type": "LV",
      "reference": "KX2010",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-20T15:00:00.000+0000",
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "4106c714-955a-435b-916c-22799b656210",
      "minLongitude": -2.0673,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.048568883650377,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T15:30:00.000+0000",
      "estimatedRestoration": "2023-05-20T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2010'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2010",
      "maxLongitude": -2.0397,
      "minLatitude": 51.4384,
      "name": "SN11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN11 9LL",
        "SN11 9LN",
        "SN11 9LW",
        "SN11 9NH",
        "SN11 9NL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0673,
              51.4504
            ],
            [
              -2.0397,
              51.4504
            ],
            [
              -2.0397,
              51.4384
            ],
            [
              -2.0673,
              51.4384
            ],
            [
              -2.0673,
              51.4504
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4504,
      "etrUpdate": true
    },
    {
      "latitude": 51.71013326630328,
      "type": "HV",
      "reference": "KX2137",
      "etaUpdate": true,
      "affectedCustomerCount": 20,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f6e09024-4378-4370-b49d-a9c5f88ec15c",
      "minLongitude": -1.3842,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3632039132642002,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T16:27:00.000+0000",
      "estimatedRestoration": "2023-05-20T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2137'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2137",
      "maxLongitude": -1.3581,
      "minLatitude": 51.6928,
      "name": "OX13 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX13 5JH",
        "OX13 5JL",
        "OX13 5JN",
        "OX13 5JR",
        "OX13 5JW",
        "OX13 5LE",
        "OX13 5LF",
        "OX13 5QW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3842,
              51.7165
            ],
            [
              -1.3581,
              51.7165
            ],
            [
              -1.3581,
              51.6928
            ],
            [
              -1.3842,
              51.6928
            ],
            [
              -1.3842,
              51.7165
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T17:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7165,
      "etrUpdate": true
    },
    {
      "latitude": 50.914589858184705,
      "type": "LV",
      "reference": "KX2170",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-20T18:15:00.000+0000",
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a6c60ab4-7a48-4164-9558-14f0380e6197",
      "minLongitude": -1.0079,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.004723845596069,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T17:20:00.000+0000",
      "estimatedRestoration": "2023-05-20T23:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2170'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2170",
      "maxLongitude": -1.0016,
      "minLatitude": 50.9131,
      "name": "PO8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO8 9NB",
        "PO8 9ND",
        "PO8 9PT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0079,
              50.9161
            ],
            [
              -1.0016,
              50.9161
            ],
            [
              -1.0016,
              50.9131
            ],
            [
              -1.0079,
              50.9131
            ],
            [
              -1.0079,
              50.9161
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T18:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9161,
      "etrUpdate": true
    },
    {
      "latitude": 50.47266370185053,
      "type": "HV",
      "reference": "KX2190",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5f109ed2-7a74-433a-80b8-f24d18cdb4b9",
      "minLongitude": -4.7513,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.746142144981664,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T18:12:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2190'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2190",
      "maxLongitude": -4.744,
      "minLatitude": 50.4691,
      "name": "PL31 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PL31 2FS",
        "PL31 2FT",
        "PL31 2FU",
        "PL31 2FW",
        "PL31 2FX",
        "PL31 2FY",
        "PL31 2FZ",
        "PL31 2GQ",
        "PL31 2QL",
        "PL31 2SB",
        "PL31 2SD",
        "PL31 2SY",
        "PL31 2TD",
        "PL31 2TG",
        "PL31 2TH",
        "PL31 2TJ",
        "PL31 2TL",
        "PL31 2TN",
        "PL31 2TP",
        "PL31 2TQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.7513,
              50.4768
            ],
            [
              -4.744,
              50.4768
            ],
            [
              -4.744,
              50.4691
            ],
            [
              -4.7513,
              50.4691
            ],
            [
              -4.7513,
              50.4768
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T21:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.4768,
      "etrUpdate": true
    },
    {
      "latitude": 50.82406909492972,
      "type": "HV",
      "reference": "KX2249",
      "etaUpdate": true,
      "affectedCustomerCount": 121,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5469481f-b3c9-4688-b602-1c6eab1757ce",
      "minLongitude": -2.3241,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.303682644681965,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T16:38:00.000+0000",
      "estimatedRestoration": "2023-05-20T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2249'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2249",
      "maxLongitude": -2.2746,
      "minLatitude": 50.7973,
      "name": "DT11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT11 0BT",
        "DT11 0BU",
        "DT11 0BX",
        "DT11 0BY",
        "DT11 0DB",
        "DT11 0DD",
        "DT11 0DE",
        "DT11 0DF",
        "DT11 0DG",
        "DT11 0DL",
        "DT11 0DQ",
        "DT11 0LQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3241,
              50.8494
            ],
            [
              -2.2746,
              50.8494
            ],
            [
              -2.2746,
              50.7973
            ],
            [
              -2.3241,
              50.7973
            ],
            [
              -2.3241,
              50.8494
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T19:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8494,
      "etrUpdate": true
    },
    {
      "latitude": 57.159885068247696,
      "type": "LV",
      "reference": "KX2246",
      "etaUpdate": true,
      "affectedCustomerCount": 27,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "b70aae20-3e76-453b-a080-9a9fe15e2c9d",
      "minLongitude": -2.1102,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.108846161316359,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T18:28:00.000+0000",
      "estimatedRestoration": "2023-05-20T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2246'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2246",
      "maxLongitude": -2.1073,
      "minLatitude": 57.1586,
      "name": "AB24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB24 3NT",
        "AB24 3NU",
        "AB24 3NW",
        "AB24 3PB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1102,
              57.1609
            ],
            [
              -2.1073,
              57.1609
            ],
            [
              -2.1073,
              57.1586
            ],
            [
              -2.1102,
              57.1586
            ],
            [
              -2.1102,
              57.1609
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T19:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.1609,
      "etrUpdate": true
    },
    {
      "latitude": 57.573547288513794,
      "type": "LV",
      "reference": "KX2181",
      "etaUpdate": true,
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "bbe39d2b-9fc6-4868-ad38-3ddffbff1419",
      "minLongitude": -4.4442,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.441296264821637,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-20T18:02:00.000+0000",
      "estimatedRestoration": "2023-05-20T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX2181'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX2181",
      "maxLongitude": -4.4303,
      "minLatitude": 57.5696,
      "name": "IV7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV7 8DF",
        "IV7 8DP",
        "IV7 8DR",
        "IV7 8DU",
        "IV7 8DW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.4442,
              57.576
            ],
            [
              -4.4303,
              57.576
            ],
            [
              -4.4303,
              57.5696
            ],
            [
              -4.4442,
              57.5696
            ],
            [
              -4.4442,
              57.576
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-20T19:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.576,
      "etrUpdate": true
    }
  ]
}
